src/app/communities/communities.service.ts

```diff
@@ -87,4 +89,10 @@ export class CommunitiesService {
           this.leaveCommunityUrlPattern.replace('{communityId}', communityId).replace('{userId}', userId));
   }
 
+  inviteUsers(communityId: string, userIds: string[]): Observable<CommunityUserRegistrationResponse[]> {
```

Comment:
I would suggest to place business logic operating on `CommunityUserRegistration` domain into a separate service called `CommunityUserRegistrationService`. There are already two methods to operate on `CommunityUserRegistration` in `CommunityUserRegistrationCommandController` (https://github.com/KnowledgeAssets/myskills-server/blob/master/src/main/java/io/knowledgeassets/myskills/server/communityuser/registration/command/CommunityUserRegistrationCommandController.java). And it is likely there will be query endpoints to fetch user registrations to display them somewhere in the web application. So we could prevent `CommunitiesService` from growing up to several hundred lines of source code by separating registration logic into a different service. @svetivanova what do you think about it?

Reply:
thank you. done